src/pudl/output/censusdp1tract.py

```diff
@@ -14,18 +14,24 @@ def census_asset_factory(layer: str) -> AssetsDefinition:
     """An asset factory for finished EIA tables."""
 
     @asset(
-        ins={"censusdp1tract_to_sqlite": AssetIn("censusdp1tract_to_sqlite")},
+        ins={
+            "raw_census__demographic_profiles_dp1": AssetIn(
+                "raw_census__demographic_profiles_dp1"
+            )
+        },
         name=f"core_censusdp1__entity_{layer}",
```

Comment:
I added `__entity_` because these tables describe entities; each row describes a single tract, state, or county.  Also these should probably use `_core` because they aren't written to the database.

Reply:
I'm not understanding the entity reasoning.

Reply:
The primary keys of these tables are GEOIDs which are unique to each tract, state and county. The fields describe the attributes of these unique entities. I view these tables as the type as `core_eia__entity_plants` or `core_eia__entity_generators`.